Index: configure.in
===================================================================
--- configure.in	(revisão 19117)
+++ configure.in	(cópia de trabalho)
@@ -119,7 +119,7 @@
 if test "x$HAVE_OGG" = "xno"
 then
   dnl fall back to the old school test
-  XIPH_PATH_OGG(, AC_MSG_ERROR(must have Ogg installed!))
+  #XIPH_PATH_OGG(, AC_MSG_ERROR(must have Ogg installed!))
   libs_save=$LIBS
   LIBS="$OGG_LIBS"
   AC_CHECK_FUNC(oggpack_writealign, , AC_MSG_ERROR(Ogg >= 1.0 required !))
